sql/core/src/test/scala/org/apache/spark/sql/streaming/FlatMapGroupsWithStateSuite.scala

```diff
@@ -54,8 +54,11 @@ class FlatMapGroupsWithStateSuite extends StateStoreMetricsTest
   import FlatMapGroupsWithStateSuite._
 
```

Comment:
Here, too. `with BeforeAndAfterAll` and `override def afterAll()` are redundant.